ts/Core/Axis/Axis.ts

```diff
@@ -7250,10 +7250,13 @@ class Axis implements AxisComposition, AxisLike {
         const isLinked = axis.isLinked;
         const ticks = axis.ticks;
         const slideInTicks = axis.chart.hasRendered && isNumber(axis.oldMin);
+        const tickInterval = axis.tickInterval;
 
         // Linked axes need an extra check to find out if
         if (!isLinked ||
-            (pos >= (axis.min as any) && pos <= (axis.max as any))
+            (pos >= (axis.min as any) && pos <= (axis.max as any)) ||
+            axis.grid &&
```

Comment:
We should avoid polluting core with product-specific code

Reply:
You are right. I will try to find another solution!

Reply:
I'm going to examine closely #13359, they might be connected.

Reply:
Thanks!